docs/getting-started/install-guide.md

```diff
@@ -6,6 +6,25 @@ This guide was last updated for v0.8.2.
 
 There are four methods for downloading and installing the Decred software. One is via dcrinstall (cross-platform), one is via the Windows Installer (Windows only of course, and the only way to get Paymetheus as of v0.8.2), another is via the precompiled binary releases (cross-platform), and the other is building the software yourself (cross-platform). The first three methods will be covered here and the fourth may be added at a later date.
 
+---
+
+## **Windows Installer for Paymetheus** 
+
+The Windows Installer (`.msi` file) is located here: [https://github.com/decred/decred-binaries/releases](https://github.com/decred/decred-binaries/releases). It will install Paymetheus to your computer's Program Files folder. Installation is pretty straightforward, but instructions are provided below:
+
+1. Download the correct file:
+
+    For 32-bit computers, download the `decred_0.8.2-beta_x86.msi` file. <br />
```

Comment:
Not really related to your PR, more of a general comment, but I hate how many places we have to specify the version.  I wonder if mkdocs lets us set a macro or variable or something for the version.